### DISCUSSION TITLE
‘CBIing’ North West lutruwita: A New Way of Seeing.

### DISCUSSION THREAD

**Discussion by lianaguegan** [life-itself/community#1087] [General]
An exciting new project is happening in lutruwita (Tasmania), Australia that is a real paradigm shifter for our region.

Founder of Live Well Tasmania, Robin Krabbe has been the key motivator in kickstarting a proposed Community Based Income (CBI) trial for north west lutruwita - something currently swelling with momentum. Robin explained that a CBI has been opted for over a Universal Basic Income (UBI) as the target group is recipients of Centrelink unemployment payments, many of whom are trauma effected. It would be unrealistic to expect the same outcomes of greater community engagement and wellbeing aligned with the unconditional payments of a UBI with this particular group. Addressing trauma needs to come first, which is what makes a CBI more appropriate in this situation.

The CBI will raise the recipients current income by $125 per week and expect a certain amount of hours per week community engagement of some kind. What makes this project so special, however, is that under the premise that well people create stronger community, engagement might simply be a few hours per week of self care! This is a major shift away from social security in Australia where unemployment is stigmatised and reward/punishment is foundational. Robin explained that in our case a CBI could function as a stepping stone to the eventual implementation of a UBI if weller people are an outcome of the trial. 

Hearteningly, a proposed CBI is gaining community attention. Both the media and local councils are indicating curiosity, and a mini trial has just been funded. This funding will help with the establishment of a Social, Health and Wellbeing Panel (SHAWP) that will identify current gaps in the health and wellbeing of the community, and co-design projects to, for example, teach disadvantaged youth health and wellbeing management skills.

A small group of 4 Centrelink ‘job seekers’ will trial the program over a 9 month period. The CBI is introducing the concept of care culture within a context of increasing social decline. This movement away from seeing trauma based behaviour such as disengagement and unemployment as something requiring stern management and punishment to something requiring care is a major transitional step here in the north west. It is a hopeful sign that better is in front of us.

**Comment by rufuspollock** [comment]
@lianaguegan this is amazing 👏 Could we get a write-up that we could post on the blog?

**Comment by lianaguegan** [comment]
Likewise, just connecting in with Live Well Tasmania https://lwt.org.au/  to see what they would recommend. Big thanks for your lovely support 🙏🏼🙏🏼The current state of the planet has arisen from the thoughts and actions of humans past and present. The ethic of peoplecare includes changing the thinking behind our actions. Once our thinking changes our actions will follow suit (Looby Macnamara)Wild Plum understands that the ancient soils of lutruwita belong to the palawa people, apologises for invasion, and works these soils with gratitudeOn 2 Jun 2024, at 8:18 AM, Rufus Pollock ***@***.***> wrote:﻿
@lianaguegan this is amazing 👏 Could we get a write-up that we could post on the blog?

**Comment by lianaguegan** [comment]
﻿﻿Dear Rufus and Lauren 🙏🏼🙏🏼Firstly, here is a write up from Robin Krabbe, co founder of Live Well Tasmania (LWT) outlining why a CBI would be transformational for North West lutruwita for the Life Itself Blog (I have cc’d Robin in). It has been a while in coming, but you will see the big lot of work that has gone into its coming into being (its formation is serving multiple purposes including contributing to the blog life of LI):Secondly, I have been thinking a lot about Life Itself lutruwita and its continuity in its present state. It sadly hasn’t really taken off in a way that I was hoping. A lot could have to do with my inexperience with running something like this, but I do think also that our lower socioeconomic demographic, despite the wonderful marketing efforts of The Reseed Centre, does not offer ripe grounding for something like a pocket of LI to flourish.I will be taking these thoughts into our next gathering this coming Sunday (which mostly is just two or three people). Having said that, I am hoping this will not result in complete closure but rather morph into something we can adapt to North West lutruwita circumstance. You will notice the strong alignment in Robin’s thoughts about the cruciality of inner development and transformation in driving sociocultural change with that of Life Itself’s theoretical stance (something I also hope to take into Sunday’s gathering). I’m feeling like Life Itself, depending on what is decided next week, could at the very least continue to function as a powerful knowledge bank for us, but perhaps we will have to find ways of practice that better suit the needs of our demographic. I will touch base with outcomes.As I’m sure you are both very aware of, and in a similar vein, the LI Australian Whats App chat has also failed to take off. I’m not sure what would need to be done to stimulate interest, but I wonder if you worry about it loosing interest instead of gaining new members? Perhaps it is not its time yet…?I’m so appreciative of all the interest, encouragement and support Life Itself has been giving us here in the North West of lutruwita. I do so hope that a relationship can be maintained and grown into something wonderful that is beneficial to us all.Warmest, lianaWild PlumThe current state of the planet has arisen from the thoughts and actions of humans past and present. The ethic of peoplecare includes changing the thinking behind our actions. Once our thinking changes our actions will follow suit (Looby Macnamara)Wild Plum understands that the ancient soils of lutruwita belong to the palawa people, apologises for invasion, and works these soils with gratitudeOn 2 Jun 2024, at 8:18 AM, Rufus Pollock ***@***.***> wrote:﻿
@lianaguegan this is amazing 👏 Could we get a write-up that we could post on the blog?
